.github/workflows/performance-analysis.yml

```
@@ -0,0 +1,65 @@
+name: "Performance analysis"
+
+on:
+  push:
+    branches: development
+  workflow_dispatch:
+
+jobs:
+  analyze:
+    name: ${{matrix.buildname}}
+    runs-on: ${{matrix.os}}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - buildname: 'pi'
+            qt: '5.14.2'
+            triplet: arm64-linux
+            compiler: gcc_64
+            os: [ self-hosted, pi4 ]
+
+
+    env:
+      # APP_BUILD embeds the git sha which changes on every commit, throwing the cache off - ignore it
+      CCACHE_IGNOREOPTIONS: "-DAPP_BUILD=*"
+
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+        submodules: true
+
+      # run-vcpkg action does not run on arm64 (https://github.com/lukka/run-vcpkg/issues/152)
+      # so install dependencies manually
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install build-essential git liblua5.1-dev zlib1g-dev libhunspell-dev libpcre3-dev \
```

Use `apt` rather than `apt-get` the former is a more modern version that "works better":
```suggestion
        sudo apt update
        sudo apt install build-essential git liblua5.1-dev zlib1g-dev libhunspell-dev libpcre3-dev \
```

---

I'm not sure it's 'better' per se, for this purpose. It seems to mostly just aggregate the most commonly used commands from apt-get and apt-cache and is a lot easier to support for an end-user, but I'm also not sure that the progress bar and other such frills will do anything good for the gha logs.

---

apt-get is better for non-interactive use: https://unix.stackexchange.com/questions/590699/should-i-use-apt-or-apt-get-in-shell-scripting#590703